scripts/action-handler.js

```diff
@@ -1467,13 +1469,21 @@ Hooks.once('tokenActionHudCoreApiReady', async (coreModule) => {
                 }
 
                 if (this.splitStrikes) {
-                    this.addActions([...auxiliaryActions, ...versatileOptionActions], strikeGroupData)
+                    if (this.showAuxiliaryActions) {
+                        this.addActions([...auxiliaryActions, ...versatileOptionActions], strikeGroupData)
+                    } else {
+                        this.addActions([...versatileOptionActions], strikeGroupData)
+                    }
```

Comment:
It should be possible to move the handling for `showAuxiliaryActions` up to where those are defined actions and just setting `auxiliaryActions` to `[]`. This'll avoid having two if/else statements and also loading the actions only for them to be ignored later on.

Reply:
Changed, it works and it's a lot cleaner.